src/core/src/enclave_sgx.rs

```diff
@@ -145,8 +145,7 @@ impl EnclaveConfig {
 
     pub fn create_signup_info(
         &mut self,
-        pub_key_hash: &Vec<u8>,
-        nonce: String,
```

Comment:
It's not removed. from signup info.
"nonce" was earlier being considered from BlockId. With new design there won't be any block available during registration for genesis node. PoET public is another unique identifier that keeps changing everytime enclave is initialized. There'll be still issue when re-registering after K block, but that is not tested yet / implemented to re-register, we can plan to add other variable component here at that time.

> Please fix commit title message length.
> And please add me as author

I will fix commit message lengths for rest of the commits.
I cherry-picked your commit, you're the only author there.

Reply:
AFAIK the new design is a temporary fix. Please do not modify the structure for short-term designs.

Reply:
Again... If  you see PoET engine new() would be initializing enclave, it can't have BlockId from service. Until service calls "start()" we will not have BlockId to be used as nonce. There's no other choice other than to modify this at this time, unless we add random string as nonce. Problem with random string is that same nonce needs to go to registration payload.